# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/povray/distribution_message.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./source/optout.h~	2005-12-17 05:02:34.000000000 +0100
+++ ./source/optout.h	2005-12-17 05:03:44.000000000 +0100
@@ -55,6 +55,7 @@
 #define OFFICIAL_VERSION_NUMBER_HEX 0x0361
 
 #define DISTRIBUTION_MESSAGE_1 "This is an unofficial version compiled by:"
+#define DISTRIBUTION_MESSAGE_2 "--- the ROCK Linux build scripts ... ( http://www.rocklinux.org ) ---"
 #ifndef DISTRIBUTION_MESSAGE_2
 #error You must complete the following DISTRIBUTION_MESSAGE_2 definition
 #define DISTRIBUTION_MESSAGE_2 " FILL IN NAME HERE........................."
